src/flask_pyoidc/flask_pyoidc.py

```diff
@@ -319,3 +342,100 @@ def valid_access_token(self, force_refresh=False):
                        id_token_jwt=response.get('id_token_jwt'),
                        refresh_token=response.get('refresh_token'))
         return access_token
+
+    def _check_authorization_header(self, request) -> bool:
+         '''Look for authorization in request header.
+         
+         Parameters
+         ----------
+         request : werkzeug.local.LocalProxy
+             flask request object
+             
+         Returns
+         -------
+         bool
+             True if the request header contains authorization.
+         '''
+         if 'Authorization' in request.headers and request.headers['Authorization'].startswith('Bearer '):
+             return True
```

Comment:
Missing `return` for the negative case.
(It will return `None` default instead of `false`.)

This could potentially be combined with `_parse_access_token`:
if a token exists in the `Authorization` header, parse it from there, otherwise return `None`.

Reply:
Earlier they were the same method but `oidc_decoraror` doesn't have to know about the token. It only needs to know if the authorization field in the request header is present or not. If it's there, `introspect_token` has the responsibility to have the token parsed from the request header and begin introspection. So I separated the responsibilities based on what is needed at what time.